src/app/components/app-node-utilizations/app-node-utilizations.component.html

```diff
@@ -15,17 +15,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  -->
-<div class="app-node-utilizations">
-  <mat-card-title class="title">Node Resource Utilization</mat-card-title>
-  <div class="mat-elevation-z8">
-    <mat-card appearance="outlined" class="box-card">
-      <mat-card-content>
-        <div class="status-wrapper flex-grid">
-          <div class="chart-wrapper flex-primary">
-            <app-vertical-bar-chart [bucketList]="bucketList" [barChartDataSets]="barChartDataSets" />
-          </div>
-        </div>
-      </mat-card-content>
-    </mat-card>
+
+ <app-card class="app-node-utilizations">
+  <ng-container header>
+    <div>
+      <i class="fa fa-chart-column" aria-hidden="true"></i>
+      Node Resource Utilization
+    </div>
+    <div>
+      <mat-form-field appearance="outline" class="partition-selector">
+        <mat-label>Partition</mat-label>
```

Comment:
Is it possible to prevent typing in this field? There will almost always be only a single partition.

Reply:
Never mind, looks like it doesn't (video made it look like it).

Reply:
Yep, sorry for the mislead. That is my browser extension feature.